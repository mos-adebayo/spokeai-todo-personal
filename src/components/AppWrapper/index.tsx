import React from "react";
import { Container } from "react-bootstrap";
import Header from "../Header";
import ErrorMessage from "../ErrorMessage";
import CreateToDo from "../TodoForm";

type Props = {
  children: React.ReactElement;
};
const AppWrapper: React.FC<Props> = ({ children }) => {
  return (
    <React.Fragment>
      <Header />

      <ErrorMessage />

      <Container>{children}</Container>

      <CreateToDo />
    </React.Fragment>
  );
};

export default AppWrapper;
