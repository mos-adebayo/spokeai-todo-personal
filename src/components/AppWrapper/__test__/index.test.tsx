import React from "react";
import { shallow } from "enzyme";
import AppWrapper from "../index";

describe("<AppWrapper />", () => {
  it("should match snapshots", () => {
    const wrapper = shallow(
      <AppWrapper>
        <p>App content</p>
      </AppWrapper>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
