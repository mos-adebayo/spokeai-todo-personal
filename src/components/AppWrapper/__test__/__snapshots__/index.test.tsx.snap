// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppWrapper /> should match snapshots 1`] = `
<Fragment>
  <Header />
  <ErrorMessage />
  <Container
    fluid={false}
  >
    <p>
      App content
    </p>
  </Container>
  <CreateToDo />
</Fragment>
`;
